ui/src/app/app.component.ts

```diff
@@ -12,6 +12,7 @@ export class AppComponent {
   sidenavComponent: string = ''
   constructor(private sidenavService: SidenavService) {}
   ngOnInit(): void {
+    localStorage.clear()
```

Comment:
few localstorage elements are being used to store a state in case user refreshes or opens a new tab so we cannot clear it from here